package.json

```diff
@@ -54,24 +56,25 @@
     "node-emoji": "^1.7.0",
     "parse-diff": "^0.4.0",
     "query-string": "^4.3.1",
-    "react": "16.0.0-alpha.12",
-    "react-native": "0.48.1",
+    "react": "^16.0.0",
+    "react-native": "^0.48.4",
     "react-native-actionsheet": "^2.2.0",
     "react-native-code-push": "^5.0.0-beta",
     "react-native-communications": "^2.2.1",
     "react-native-config": "^0.6.0",
     "react-native-cookies": "^3.2.0",
     "react-native-device-info": "^0.11.0",
-    "react-native-elements": "^0.9.3",
+    "react-native-elements": "^0.17.0",
     "react-native-htmlview": "^0.12.0",
     "react-native-i18n": "^2.0.4",
     "react-native-material-design-searchbar": "^1.1.4",
-    "react-native-parallax-scroll-view": "^0.19.0",
+    "react-native-mock": "https://github.com/shqld/react-native-mock/tarball/master",
+    "react-native-parallax-scroll-view": "https://github.com/mini-eggs/react-native-parallax-scroll-view/tarball/react16-support",
     "react-native-photo-view": "^1.5.2",
     "react-native-safari-view": "^2.0.0",
-    "react-native-search-bar": "^3.0.0",
+    "react-native-search-bar": "https://github.com/andrewda/react-native-search-bar/tarball/master",
     "react-native-syntax-highlighter": "^1.2.1",
-    "react-native-table-component": "^1.0.5",
+    "react-native-table-component": "https://github.com/jaynakus/react-native-table-component/tarball/master",
```

Comment:
Can be set to `^1.1.0` :tada:

Reply:
Why is this pointing to github instead of a regular version number?

Reply:
This is the fork that works with React 16. Now 1.1.0 works, too

Reply:
Oh, makes sense! Thanks for clarification @andrewda!